packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -5358,9 +5355,9 @@ void main() {
           // Left padding is 12 per Material 3 spec.
           expect(tester.getRect(findPrefixIconInnerRichText()).left, 12.0);
           // Check the padding between the prefix icon and the input.
-          // TODO(bleroux): the gap between the icon and the input should be 16 based on M3 specification.
+          // The gap between the icon and the input should be 16 based on M3 specification.
           // See https://github.com/flutter/flutter/issues/149408.
```

Comment:
Remove this reference to the issue, this link was required for the TODO (it no more required now).